src/main/java/org/frankframework/frankdoc/model/FrankAttribute.java

```diff
@@ -67,11 +67,7 @@ public Key getKey() {
 
 	@Override
 	boolean overrideIsMeaningful(ElementChild overriddenFrom) {
-		// There is no need to check here for a change of the mandatory field. If the inherited
-		// attribute has different mandatory or optional features, then it already gets
-		// documented=true. This is enough to have the attribute again in the declared
-		// attribute group of the child.
-		return false;
+		return isMandatory() != overriddenFrom.isMandatory();
```

Comment:
Dit zou niet alleen moeten gelden voor mandatory, maar ook voor alle andere features.

Reply:
Ik heb alle member variabelen die er moeten zijn erin gezet. Ik heb deze implementatie ook verplaatst naar ElementChild voor zover mogelijk. ElementChild.overrideIsMeaningful() gebruikt alleen de velden van ElementChild. Deze method delegeert naar de abstracte method specificOverrideIsMeaningful() om de velden van FrankAttribute of ConfigChild te controleren.

Reply:
Als je een veld toevoegt in ElementChild, dan moet ElementChild.overrideIsMeaningful() mogelijk aanpassen. Als je een veld toevoegt in FrankAttribute, dan met je daar specificOverrideIsMeaningful() aanpassen en idem voor ConfigChild. Als je een veld toevoegt aan een van deze drie classes, dan moet je de rest van de class lezen om te zien dat je de genoemde methods misschien moet bijwerken. Maar je hoeft niet in andere classes te kijken om dit te zien.